docs/articles/documentation/test-api/A-Z.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: docs
+title: A-Z
+permalink: /documentation/test-api/a-z.html
+---
+# A-Z
+
+This topic contains an alphabetical list of all the test API functions and objects with members.
+
+## Functions
+
+* [afterEach](test-code-structure.html#initialization-and-clean-up)
+* [beforeEach](test-code-structure.html#initialization-and-clean-up)
+* [fixture](test-code-structure.html#fixtures)
+* [page](test-code-structure.html#specifying-the-start-webpage)
+* [test](test-code-structure.html#tests)
```

Comment:
@inikulin Can we introduce the `Fixture` object to the documentation? Currently, when we describe `page`, `afterEach` and `beforeEach`, we use the _fixture declaration_ term.